src/controllers/article.js

```diff
@@ -36,3 +37,25 @@ export const UpdateArticle = async () => true;
  * @returns {int} Returns true after deleting an article
  */
 export const DeleteArticle = async () => true;
+/**
+ * @class article
+ */
+
+/**
+   *
+   * @name commentOnHighlight
+   * @param {req} req - Request object
+   * @param {res} res - Response object
+   * @description This method add comment for highlighted text of an article
+   * @returns {int} Returns the comment after success
+   */
+export const commentOnHighlight = async (req, res) => {
+  try {
+    const { userId, articleId, highlightedText, comment } = req.body;
+    const highlightComment = await Highlight.create({ userId, articleId, highlightedText, comment,
```

Comment:
@Oluwaseyi000 what if we only keep the index of the text highlighted i.e where the highlight start and where it stops that way we don't keep the text and it will make the functionality more accurate